guides/micronaut-scope-types/kotlin/src/main/kotlin/example/micronaut/singleton/RobotFather.kt

```diff
@@ -0,0 +1,10 @@
+package example.micronaut.singleton
+
+import jakarta.inject.Singleton
+
+@Singleton // <1>
+class RobotFather(private val robot: Robot) { // <2>
+    fun child(): Robot {
```

Comment:
should not we do `fun child() = robot` instead?

Reply:
oops, fixed 😄